Translate/En/21-08.md

```diff
@@ -1,118 +1,121 @@
+However, while we are trying to solve this phenomenon, we must first defend as many Underworld people as possible.
 
-Asuna stood up with the help of Solterina, inspiring the exhausted spirit and confirmed the circumstances.
+Asuna stood up with the help of Solterina, inspiring the exhausted spirit, as she assess the circumstances.
 
-And I felt something new despair came into my mind as it turned black and cold water.
+And I felt something new, and full of despair came into my mind, like cold dark water pouring in.
 
-More than 80% of players in China and Korea over ten thousand have already entered battle between ally. Even if it is said, the morale difference, the main faction is overwhelming the other players. Avatar disappearing here and there Blue light pillar of effect rises, every time a ferocious scream echoes.
+More than 80% of players in China and Korea, over ten thousand players have already entered the battle between allies. Even if it is said, the morale difference, the main faction is still overwhelimg the resisting factions. As Avatar's disappear here and there, effects in a pillar of blue light rises, and every time, a ferocious scream echoes.
 
-In addition, some of the main factions ……There is little battle capability left for the Japanese, and the underworld people including the integrity knight Renri also are full of creation. Even if it says that there is an advantage of sacred art and sword skill, I do not think it can be repulsed very much.
+In addition, some of the moderate factions…… There is little battle capability left in the Japanese, and the underworld people including the integrity knight Renri also which are all on their last breath(??). Even if there is an advantage from the possession of sacred art and sword skills, I do not think the opposition can be repulsed much.
 
-I can not find any words anymore, so I can only sit in the arm of Solterina.
+I cannot find any words anymore, I can only stand up while gripping Solterina's arm.
 
-With several echoes Loud laughter of the POH has arrived..
+Echoes of loud laughter from PoH arrives...
 
-Beside the lying Kirito, the death god who left a large hole in his chest raised his right hand and a five-finger extended left hand holding the giant friendship kitchen knife at the highest and laughed with his upper body rebelled against him . One day the black clouds are huge swirls in the sky, resources of life which is dissipated in the battlefield are hung down on the tornado and are sucked into the body of POH.
+Beside Kirito, who was lying down, the death god who has a large hole in his chest raised his right hand and all his five-finger extended and his left hand, which is holding the giant friendship kitchen knife at its highest and laughed with his upper body resisting its weight. Huge black clouds swirls in the sky at daytime, a funnel of life resources dissipated in the battlefield forms a tornado as they all are absorbed into PoH's body.
```

Comment:
BTW, Defan had a proper name for that knife, something like 'Knife for your friend' but not sure.